tests/e2e-tests/specs/backend/cart.test.js

```diff
@@ -3,28 +3,53 @@
  */
 import {
 	insertBlock,
-	getEditedPostContent,
 	getAllBlocks,
-	createNewPost,
 	switchUserToAdmin,
 } from '@wordpress/e2e-test-utils';
 
-describe( 'Cart Block', () => {
-	beforeEach( async () => {
-		await switchUserToAdmin();
-		await createNewPost();
-	} );
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
```

Comment:
looks like this isn't needed because there are no internal dependencies? This applies to other specs as well, I suspect it's cruft from when you switched to using the `@woocommerce/blocks-test-utils` alias?